components/pages/acknowledgements/constants.tsx

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
don't use `constants` as file name. these are not constants. convention gives this meaning. 

constants are usually primitives not expected to change - magic numbers, config values, "enum" style objects - think about it, we're changing the content in these PRs, they're not constants.

rename to `policies.tsx`
note the lower case, because it's data. follows the flow of.... "here is a module that has data in it called `policies`, it exports `policy` objects to be used in `Policy` component". very domain driven ;)

Look how readable and comprehensible the import becomes: 
`import { iMicroSeqPolicy } from './policies';` 
vs 
`import { iMicroSeqPolicy } from './constants';`

Reply:
Updated, w/ export change as well